src/base/quda_bridge.cpp

```diff
@@ -547,16 +549,16 @@ namespace quda_iface
 	  inv_param.matpc_type=QUDA_MATPC_EVEN_EVEN;
 	
 	inv_param.inv_type=QUDA_GCR_INVERTER;
-	inv_param.gcrNkrylov=10; //from default in read_input.l
+	inv_param.gcrNkrylov=24;// 10; //from default in read_input.l
 	inv_param.inv_type_precondition=QUDA_MG_INVERTER;
 	inv_param.schwarz_type=QUDA_ADDITIVE_SCHWARZ;
-	inv_param.reliable_delta=1e-10;
-	inv_param.reliable_delta_refinement=1e-10;
+	inv_param.reliable_delta=0.01;//1e-10;
+	inv_param.reliable_delta_refinement=0.0001;//1e-10;
 	inv_param.precondition_cycle=1;
 	inv_param.tol_precondition=1e-1;
 	inv_param.maxiter_precondition=1;
 	inv_param.gamma_basis=QUDA_CHIRAL_GAMMA_BASIS;
-	inv_param.solve_type=QUDA_DIRECT_SOLVE;
+	inv_param.solve_type=QUDA_DIRECT_PC_SOLVE;//QUDA_DIRECT_SOLVE;
```

Comment:
this depends on whether you're doing an EO-precon solve or not

in general one can say that one can have the outer solver doing the direct solve while doing preconditioning on the interior solves

Reply:
I think that is set up in
https://github.com/sunpho84/nissa/blob/65a62d37df5dcc8665ed1256b885197d9a003e53/src/base/quda_bridge.cpp#L570
at least this is what I understood matches tmLQCD.